config/environments/development.rb

```diff
@@ -9,35 +9,32 @@
   # Prepare the ingress controller used to receive mail
   config.action_mailbox.ingress = :sendgrid
 
-  # In the development environment your application's code is reloaded any time
-  # it changes. This slows down response time but is perfect for development
-  # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  # Make code changes take effect immediately without server restart.
+  config.enable_reloading = true
```

Comment:
`cache_classes` got flipped and changed to `enable_reloading` https://github.com/rails/spring/blob/0b8dfae4ba95a635856b3dc32495ccce6b513241/README.md?plain=1#L74-L80